app/uploaders/base_uploader.rb

```diff
@@ -26,6 +26,8 @@ def size_range
 
   # strip EXIF (and GPS) data
   def strip_exif
+    return if file.content_type.include?("svg")
```

Comment:
There will be no exif data for an SVG

Reply:
(nonblocker): This seems like a good inline comment for the code.

Reply:
✅